_posts/pl/2021-02-01-wszystko-co-chcielibyscie-wiedziec-o-fontach-ale-baliscie-sie-zapytac.md

```diff
@@ -0,0 +1,104 @@
+---
+layout:    post
+title:     Wszystko, co chcielibyście wiedzieć o fontach, ale baliście się zapytać
+published: false
+date:      2021-01-01 09:00:00 +0100
+author:    pgrobelny
+lang:      pl
+image:     
+tags:
+    - font
+    - czcionka
+    - rem
+    - woff
+    - font-face
+---
+
+Większość osób, które pracowały na frontendzie na pewno miały do czynienia z fontami, czy to wrzucały je do projektu,
```

Comment:
Jeśli "większość osób", to "miała do czynienia" i "zmieniała sposób", bo "większość" to ona :)